apps/glusterfs/volume_entry_allocate.go

```diff
@@ -131,6 +131,29 @@ func (v *VolumeEntry) replaceBrickInVolume(db *bolt.DB, executor executors.Execu
 		return err
 	}
 
+	// Get self heal status for this brick's volume
+	healinfo, err := executor.HealInfo(oldBrickNodeEntry.ManageHostName(), v.Info.Name)
```

Comment:
Getting heal info only from one node may be not sufficient, cant we have a fallback ?